src/stores/commitLog.js

```diff
@@ -24,7 +29,42 @@ export const commitLogStore = defineStore('commitLog', {
       this.selectedCommitHash = hash;
     },
     clearSelectedCommitHash: function () {
-      this.selectedCommitHash = '';
+      this.selectedCommitHash = UNCOMMITED;
+    },
+    setUncommitedFilesAmount: function (amount) {
+      this.uncommitedFilesAmount = amount || 0;
+    }
+  },
+  getters: {
+    ...mapState(commitsStore, [
+      'commits'
+    ]),
+    selectedCommitIndex: function () {
+      if (this.selectedCommitHash) {
+        const commitIndex = this.commits.findIndex((commit) => {
+          return commit.hash === this.selectedCommitHash;
+        });
+        return commitIndex;
+      }
+      return -1;
+    },
+    firstEverCommitSelected: function () {
```

Comment:
```suggestion
    // Once we stop showing all commits and only show those for the current branch, this will stop working.
    firstEverCommitSelected: function () {
```